sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -1,7 +1,302 @@
-# azure_messaging_servicebus
+<!-- cspell:words eventhubs  -->
 
-Azure Event Hubs crate for the Microsoft Azure SDK for Rust.
+# Azure Event Hubs Client Package for Rust
 
-This crate is part of a collection of crates: for more information please refer to [https://github.com/azure/azure-sdk-for-rust](https://github.com/azure/azure-sdk-for-rust).
+Azure Event Hubs crate for the Microsoft Azure SDK for Rust.
 
 License: MIT
```

Comment:
License is already at the bottom with the usual block. I don't recall seeing it this high or this basic in any other language. Would just remove.